packages/xfce4_terminal.rb

```diff
@@ -36,5 +37,13 @@ def self.build
 
   def self.install
     system "make install DESTDIR=#{CREW_DEST_DIR}"
+    system "cat <<'EOF'> xfce4-terminal
+WAYLAND_DISPLAY=wayland-0
+GDK_BACKEND=wayland
+DISPLAY=
+#{CREW_PREFIX}/bin/xfce4-terminal $@
+EOF"
+    FileUtils.mv "#{CREW_DEST_PREFIX}/bin/xfce4-terminal", "#{CREW_DEST_PREFIX}/bin/xfce4_terminal"
+    system "install -Dm755 xfce4-terminal #{CREW_DEST_PREFIX}/bin/xfce4-terminal"
```

Comment:
So we end up with `xfce4_terminal` and `xfce4-terminal`?  What is the difference?

Reply:
The new `xfce4-terminal` script will set the Wayland display variable at start